packages/expo-updates/android/src/androidTest/java/expo/modules/updates/loader/CryptoTest.kt

```diff
@@ -94,4 +97,48 @@ class CryptoTest {
     val codesigningInfo = Crypto.parseSignatureHeader("sig=\"aGVsbG8=\", keyid=\"other\"")
     Crypto.isSignatureValid(codeSigningConfiguration, codesigningInfo, testBody.toByteArray())
   }
+
+  @Test(expected = Exception::class)
+  @Throws(Exception::class)
+  fun test_construct_multipleCertsInPEMThrowsError() {
+    Crypto.CodeSigningConfiguration(
+      testCertificate + testCertificate,
+      mapOf(
+        Crypto.CODE_SIGNING_METADATA_KEY_ID_KEY to "test"
+      )
+    ).embeddedCertificate
+  }
+
+  @Test(expected = Exception::class)
+  @Throws(Exception::class)
```

Comment:
Reduce the chance that a broken method causes this test to pass
```suggestion
  @Test(expected = CertificateExpiredException::class)
  @Throws(CertificateExpiredException::class)
```